src/po/main.pot

```diff
@@ -0,0 +1,569 @@
+# The main translation file for the openblok software.
+# Copyright (C) 2021 Igor Skoric
+# This file is distributed under the same license as the openblok package.
+# Igor Skoric <igor@skoric.at>, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2021-06-25 14:09+0200\n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
```

Comment:
This is an autogenerated file. For me, xgettext generates the following header:

```
# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-06-29 16:51+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"
```

Please use this without changes here. You can, of course, customize it in the German one.

Reply:
Ok, sure.